jenkins/opensearch/perf-test.jenkinsfile

```diff
@@ -28,14 +40,22 @@ pipeline {
             H 17 * * * %BUNDLE_MANIFEST_URL=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.4.0/latest/linux/arm64/tar/dist/opensearch/manifest.yml;TEST_ITERATIONS=250;WARMUP_ITERATIONS=0
```

Comment:
I think we can remove older versions (i.e <2.5.0 and 1.3.8)

Reply:
Thank you @bbarani and @peterzhuamazon ... cool .. let me remove those already out ...